test/dynamo/test_structured_trace.py

```diff
@@ -136,6 +144,9 @@ def test_schedule(self):
             self.buffer.getvalue(),
             """\
 {"dynamo_start": {"stack": "STACK"}, "frame_id": 0, "frame_compile_id": 0, "attempt": 0}
+{"describe_storage": {"id": 0, "describer_id": "ID", "size": 4000000}, "frame_id": 0, "frame_compile_id": 0, "attempt": 0}
```

Comment:
Which field distinguishes between different tensor trace logs in a complicated recursive describe? I.e. if the call stack goes like

describe_tensor  (outer_tensor) -> describe_tensor(inner_tensor1) -> describe_storage(inner_tensor1) -> describe_storage(outer_tensor)

is it `id`? Just thinking ahead on how we'll render this in tlparse

Reply:
it's the ids. There are separate ids for tensor and storage, and they form a DAG which tlparse will have to relink